commands/sync_megaphone.py

```diff
@@ -46,25 +43,12 @@ def send_version(self, version):
         )
 
     def get_version(self):
-        ws_url = f"wss://{self.host}"
-
-        async def _get_version():
-            async with websockets.connect(ws_url) as websocket:
-                logging.info(f"Send hello handshake to {ws_url}")
-                data = {
-                    "messageType": "hello",
-                    "broadcasts": {self.broadcaster_id: "v0"},
-                    "use_webpush": True,
-                }
-                await websocket.send(json.dumps(data))
-                body = await websocket.recv()
-                response = json.loads(body)
-
-            etag = response["broadcasts"][self.broadcaster_id]
-            return etag[1:-1]  # strip quotes.
-
-        # async to sync
-        return asyncio.run(_get_version())
+        url = f"{self.url}/broadcasts"
+        resp = requests.get(url, auth=self.auth)
+        resp.raise_for_status()
+        broadcasts = resp.json()
+        etag = broadcasts["broadcasts"][self.broadcaster_id]
+        return etag[1:-1]  # strip quotes.
```

Comment:
Can we make that `etag.strip('"')` instead? Simply removing the first and last character probably works, but makes me nervous.

Reply:
You're right! Plus it makes the comment superfluous!